### PR TITLE
Completed Bootcamp

### PR DESCRIPTION
**Pull request by Garrett-Miller-Junk**
PNG file (BootCampResults.png) was added to show the final graph and accuracy (72.34% accuracy). I apologize for one big commit instead of continual changes. Please let me know of any changes I should make.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,17 +1,63 @@
...
+# import tensorflow for creating the neural network
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras.layers import Conv2D, MaxPooling2D, Flatten, Dense
+
+# use keras to import the CIFAR-10 dataset
+from keras.datasets import cifar10
+
+# number of Epochs when training the neural network. (can be personally modified before running)
+NUM_EPOCHS = 150
```
Comment (Shrinjay) [inline main.py:13]: That is a lot of epochs, I'd love whatever supercomputer you used to train this

**Comment by Shrinjay** [comment]
Looks good to me, welcome aboard!
